core/src/main/java/com/backblaze/b2/client/B2AccountAuthorizerSimpleImpl.java

```diff
@@ -14,43 +14,43 @@
  * Almost everyone is expected to use this right now.
  */
 public class B2AccountAuthorizerSimpleImpl implements B2AccountAuthorizer {
-    private final String accountId;
+    private final String applicationKeyId;
```

Comment:
the claim is that changing this does NOT change the API nor does it change what we send on the wire, right?